tests/sample_radius/migrations/0003_radius_token.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2 on 2019-06-07 07:32
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('sample_radius', '0002_default_radius_groups'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='nas',
+            name='type',
+            field=models.CharField(choices=[('cisco', 'Cisco Router'), ('Async', 'Async'), ('Sync', 'Sync'), ('ISDN Sync', 'ISDN Sync'), ('ISDN Async V.120', 'ISDN Async V.120'), ('ISDN Async V.110', 'ISDN Async V.110'), ('Virtual', 'Virtual'), ('PIAFS', 'PIAFS'), ('HDLC Clear', 'HDLC Clear'), ('Channel', 'Channel'), ('X.25', 'X.25'), ('X.75', 'X.75'), ('G.3 Fax', 'G.3 Fax'), ('SDSL', 'SDSL - Symmetric DSL'), ('ADSL-CAP', 'ADSL-CAP'), ('ADSL-DMT', 'ADSL-DMT'), ('IDSL', 'IDSL'), ('Ethernet', 'Ethernet'), ('xDSL', 'xDSL'), ('Cable', 'Cable'), ('Wireless - Other', 'Wireless - Other'), ('IEEE 802.11', 'Wireless - IEEE 802.11'), ('Token-Ring', 'Token-Ring'), ('FDDI', 'FDDI'), ('Wireless - CDMA2000', 'Wireless - CDMA2000'), ('Wireless - UMTS', 'Wireless - UMTS'), ('Wireless - 1X-EV', 'Wireless - 1X-EV'), ('IAPP', 'IAPP'), ('FTTP', 'FTTP'), ('IEEE 802.16', 'Wireless - IEEE 802.16'), ('IEEE 802.20', 'Wireless - IEEE 802.20'), ('IEEE 802.22', 'Wireless - IEEE 802.22'), ('PPPoA', 'PPPoA - PPP over ATM'), ('PPPoEoA', 'PPPoEoA - PPP over Ethernet over ATM'), ('PPPoEoE', 'PPPoEoE - PPP over Ethernet over Ethernet'), ('PPPoEoVLAN', 'PPPoEoVLAN - PPP over Ethernet over VLAN'), ('PPPoEoQinQ', 'PPPoEoQinQ - PPP over Ethernet over IEEE 802.1QinQ'), ('xPON', 'xPON - Passive Optical Network'), ('Wireless - XGP', 'Wireless - XGP'), ('WiMAX', ' WiMAX Pre-Release 8 IWK Function'), ('WIMAX-WIFI-IWK', 'WIMAX-WIFI-IWK: WiMAX WIFI Interworking'), ('WIMAX-SFF', 'WIMAX-SFF: Signaling Forwarding Function for LTE/3GPP2'), ('WIMAX-HA-LMA', 'WIMAX-HA-LMA: WiMAX HA and or LMA function'), ('WIMAX-DHCP', 'WIMAX-DHCP: WIMAX DCHP service'), ('WIMAX-LBS', 'WIMAX-LBS: WiMAX location based service'), ('WIMAX-WVS', 'WIMAX-WVS: WiMAX voice service'), ('Other', 'Other')], default='other', max_length=30, verbose_name='type'),
```

Comment:
this is weird, why is this data migration here?
If this change is necessary at all, I would prefer if it's performed separately. What do you think?

Reply:
Yes, that should be performed separately. I think someone didn't migrate the sample_radius models, last time he made changes to the models, that's why it is there. I'll separate both the changes.